lib/semantic-toast.jsx

```diff
@@ -14,14 +14,20 @@ const icons = {
     warning: 'warning circle'
 };
 
+const onDismiss = (e, onClose) => {
+    e.stopPropagation();
```

Comment:
Can you explain your reasoning on calling `stopPropagation()` here but not on the `onClick()` handler?

Reply:
When the user clicks the close button, then without `stopPropagation ()` the `onClick` event will be raised.

Reply:
So what you are saying is that it does not make sense to generate an `onClick` event for closing the toast, but it does make sense to generate one if the user clicks on it, correct? I think I agree with you.

Reply:
Yeah, right. We don't want to get `onClick` when we close the toast